readme.txt

```diff
@@ -304,6 +304,7 @@ Yes! We're offering a [custom plugin development](https://bracketspace.com/custo
 
 = [Next] =
 
+* [Changed] Global functions moved to namespace and set as deprecated.
```

Comment:
There were quite a few changes to those functions. Please list them below in **Compatibility Breaking Changes** section, in this format:

* Renamed functions:
`notification_adapt()` -> `BracketSpace\Notification\adaptNotification()`
...